pkg/commands/git.go

```diff
@@ -441,6 +441,12 @@ func (c *GitCommand) Commit(message string, flags string) (*exec.Cmd, error) {
 	return nil, c.OSCommand.RunCommand(command)
 }
 
+// Get the subject of the HEAD commit
+func (c *GitCommand) GetHeadCommitMessage() (string, error) {
+	message, err := c.OSCommand.RunCommandWithOutput("git log -1 --pretty='%s'")
```

Comment:
You can try changing `"` to ```

Reply:
Or `%s` to `%gs`

Reply:
``` ` ``` does not seem to help and `git log -1 --pretty=%gs` gives an empty string.